src/components/themes/common/overrides/TextField.ts

```diff
@@ -1,16 +0,0 @@
-import { CustomComponents } from '../../types'
-
-export default function TextField(): CustomComponents {
-  return {
-    MuiFormHelperText: {
-      styleOverrides: {
```

Comment:
we can still override this, but only to reduce its height.
I suggest the following overrides:
```
root: {
          '&.Mui-error': {
            marginTop: 0,
            lineHeight: 1
          }
        }
```